### ISSUE TITLE
Change donations to take a single card instead of a list

### ISSUE THREAD

**Issue by JoshSmith** [code-corps/code-corps-ember#783]
# Problem

First, we need to take the design changes from #703 and narrow them down to styling of the individual card only.

If the user has a card:
- [x] show the card item
- [x] hit donate
- _NOT MVP - TODO LATER_ you can edit or replace the card
> Visa 4242 [edit] [replace]
[edit] lets you update expiration dates
[replace] gives you card form with cancel link or donate button

If the user doesn't have a card:
- [x] show the new card form
- [ ] hit donate
- [ ] creates customer (if needed)
- [ ] creates card
- [ ] creates subscription

**Comment by rlueder** [comment]
I'll work on this one.

**Comment by rlueder** [comment]
@JoshSmith: most of what's needed seems to be in place already with only a few small tweaks needed, here's my understanding of the task:

1. rename "Add Card" button to "Donate";

2. on donation page change header from "Payment information" to "Review Payment", I'll also review the styles since there's no concept of selecting a card anymore;

3. I'll also consider the Edit and Replace buttons, it won't take me much longer for me to style the static buttons. We can hide the elements and add links to them on a separate ticket;

### Future "Nice-to-Have"

- Sign Up should be the default option for an anonymous user (with Sign In as an option on the page itself and not just main nav "already registered? Sign In to donate");

- if a user had previously decided to donate by clicking on "Become a Donor" and then choosing a specific dollar amount it'd be nice to take them directly to the "Payment" page (formerly "Add a Card") with that amount already selected ($xx given each month) so they don't have to repeat the steps after signing up;

- allow users to donate without signing up, ask for an email with payment information and send them into selecting interests, expertise and skills only after they've already donated (direct Sign Up flow is not affected)

**Comment by JoshSmith** [comment]
Edit and replace have difficult actions associated with them, so if we can leave them out for now and circle back later, I think that's better.

**Comment by JoshSmith** [comment]
I'm not sure that there should be a duplicated "Donate" button.

I think there's more here than what you listed above, for example we need to remove the cancel link. `canCancel` similar is not a thing.

When the card gets added, we also probably don't want it to be displaying the card, which I imagine it might do when that promise resolves.

**Comment by JoshSmith** [comment]
Closed by #793.
